### PR TITLE
Add support for Ubuntu 18.04 (bionic)

### PR DESCRIPTION
**Pull request by scollazo**
This connects to archivematica/Issues#119.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@
   pause:
     seconds: 10
 
-- name: "check clamdscan is working"
-  command: "clamdscan /bin/sh"
+#- name: "check clamdscan is working"
+#  command: "clamdscan /bin/sh"
```
Comment (hakamine) [inline tasks/main.yml]: Why remove the "check clamdscan is working" task?

**Review by hakamine — CHANGES_REQUESTED** [review]
Why is it needed to remove/comment out the task that checks if clamdscan is working?

**Review by mamedin — APPROVED** [review]
LGTM

File: tasks/main.yml (hunk 1 above)
Comment (scollazo) [reply tasks/main.yml]: That was an ad-hoc change at first, but it shouldn't have reached github.

**Comment by scollazo** [comment]
@hakamine the clamd check is back

**Pull request merged**